cli/cmd/test_helper.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/sergi/go-diff/diffmatchpatch"
+)
+
+func diffCompare(t *testing.T, actual string, expected string) {
+	if actual != expected {
+		dmp := diffmatchpatch.New()
+		diffs := dmp.DiffMain(actual, expected, true)
+		patches := dmp.PatchMake(expected, diffs)
+		patchText := dmp.PatchToText(patches)
+		t.Fatalf("Unexpected output:\n%+v", patchText)
+
```

Comment:
NIT: The extra blank line here should be removed.

Reply:
Oops sorry about that.